packages/react-fresh/src/__tests__/__snapshots__/ReactFreshBabelPlugin-test.js.snap

```diff
@@ -1,3 +1,184 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ReactFreshBabelPlugin hello world 1`] = `"hello();"`;
+exports[`ReactFreshBabelPlugin ignores HOC definitions 1`] = `
+"
+let connect = () => {
+  function Comp() {
+    const handleClick = () => {};
+    return <h1 onClick={handleClick}>Hi</h1>;
+  }
+  return Comp;
+};
+function withRouter() {
+  return function Child() {
+    const handleClick = () => {};
+    return <h1 onClick={handleClick}>Hi</h1>;
+  };
+};"
+`;
+
+exports[`ReactFreshBabelPlugin ignores complex definitions 1`] = `
+"
+let A = foo ? () => {
+  return <h1>Hi</h1>;
+} : null;
+const B = function Foo() {
+  return <h1>Hi</h1>;
+}();
+let C = () => () => {
+  return <h1>Hi</h1>;
+};
+let D = bar && (() => {
+  return <h1>Hi</h1>;
+});"
+`;
+
+exports[`ReactFreshBabelPlugin only registers pascal case functions 1`] = `
+"
+function hello() {
+  return 2 * 2;
+}"
+`;
+
+exports[`ReactFreshBabelPlugin registers top-level exported function declarations 1`] = `
+"
+export function Hello() {
+  function handleClick() {}
+  return <h1 onClick={handleClick}>Hi</h1>;
+}
+
+_c = Hello
+export default function Bar() {
+  return <Hello />;
+}
+
+_c2 = Bar
+function Baz() {
+  return <h1>OK</h1>;
+}
+
+_c3 = Baz
+const NotAComp = 'hi';
+export { Baz, NotAComp };
+
+export function sum() {}
+export const Bad = 42;
+
+var _c;
+
+var _c2;
+
+var _c3;
+
+__register__(_c, 'Hello');
```

Comment:
Wonder why it sometimes uses single quotes and other times uses double quotes with escape slashes...